includes/class-wc-gateway-ppec-cart-handler.php

```diff
@@ -70,8 +70,8 @@ public function wc_ajax_generate_cart() {
 		WC()->shipping->reset_shipping();
 		$product = wc_get_product( $post->ID );
```

Comment:
I don't get why this is needed, that's what threw me off. It looks like this code is half-prepared for the case where the user doesn't have JS enabled, but it clearly can't happen.